test/unit/test_bam_pa_tag_parity.cpp

```
@@ -0,0 +1,340 @@
+}
+
+// Index of `name` in `names`, or -1.
+int index_of(const std::vector<std::string> &names, const std::string &name) {
+    for (size_t i = 0; i < names.size(); ++i)
+        if (names[i] == name) return (int)i;
+    return -1;
+}
+
+}  // namespace
+
+TEST_CASE("pa: the --bam value is the float32 of the SAM text token") {
```

Cppcheck (2.21.0)

[style] The function 'count_data' is never used.

(unusedFunction)